
syntax = "proto3";

message OnvifCliDeviceParam {
	string strName = 1;
	string strPasswd = 2;
	
	//Url of the onvif device endpoint
	string strUrl = 3;
}

message OnvifCliDeviceLoginReq {
	string strUserName = 1;
	string strPasswd = 2;
	
	//Url of the onvif device endpoin
	string strUrl = 3;
}

message OnvifCliDeviceLoginResp {
	string strHandle = 1;
	bool bLogined = 2;
}


